commands/serve/lib/webpack.build.js

```diff
@@ -86,6 +86,7 @@ const cfg = ({ srcDir, distDir, dev = false, serveConfig = {} }) => {
         __NEBULA_DEV__: true,
         'process.env.NEBULA_VERSION': JSON.stringify(version),
         'process.env.NEBULA_VERSION_HASH': JSON.stringify(versionHash),
+        ...(typeof serveConfig.replacementStrings === 'function' ? serveConfig.replacementStrings() : {}),
```

Comment:
why is this needed? this looks to only compile the nebula serve page not the supernova used

Reply:
You are right, this is absolutely not needed. I misunderstood the code somehow... Removed!